distribution/kernel/src/main/descriptors/bin.xml

```diff
@@ -13,25 +13,14 @@
 -->
 <assembly>
 
-  <id>bin</id>
+    <id>bin</id>
 
-  <formats>
-    <format>zip</format>
-  </formats>
+    <formats>
+        <format>zip</format>
+    </formats>
 
- 
     <componentDescriptors>
         <componentDescriptor>target/dependencies/common-bin.xml</componentDescriptor>
     </componentDescriptors>
-      
-  <files>
-        <file>
-        <!-- Install the features descriptor into the system repository -->
-            <source>${basedir}/target/classes/${artifactId}-features.xml</source>
-            <outputDirectory>system/ddf/distribution/${artifactId}/${version}</outputDirectory>
```

Comment:
❓ Is there a particular reason this was needed previously and why it's no longer needed?

Reply:
This section used to install the `features.xml` file into the `/system` directory in the zip. This PR deleted that `features.xml` file that used to be in this module, so this section is no longer necessary.

Reply:
Sorry I didn't word that very well. I guess what I'm wondering is if the kernel features file needed to be installed in the system directory before, why doesn't the platform features file need to be installed in system now that it has the "kernel" feature? Maybe this just hasn't been needed for a long time now. I was just curious :)

Reply:
This section is included in the example for a custom distribution in the Documentation: https://karaf.apache.org/manual/latest/ (search for `src/main/descriptors/bin.xml`).

Reply:
👍  That makes sense.